### PR TITLE
Added conditional sros env variables to the drone setup script

### PR DESCRIPTION
**Pull request by quinz**

### CODE REVIEW

**Review by jlaitine — COMMENTED** [review]
This looks fine starting point to me;
only thing is, that the ros nodes are being executed as user, so they don't probably have write access to /enclave, so log directory needs to be somewhere else? Just let me know when you have tested these on a real drone!

**Review by jlaitine — COMMENTED** [review]
Maybe put the sros config into a seprate file and source it in here, the same as drone device id; but this is not obligatory

File: packaging/systemd/setup_fog.sh [hunk 1]
```diff
@@ -6,5 +6,18 @@
...
 
+# Enable SROS
+DIR="/enclave/sros"
+if [ -d "$DIR" ]; then
+  echo "Folder ${DIR} exists. Enabling SROS2."
+  export ROS_SECURITY_ENABLE=true
+  export ROS_SECURITY_STRATEGY=Enforce
+  export ROS_SECURITY_KEYSTORE="${DIR}"
+  export ROS_SECURITY_LOG_VERBOSITY=WARN
+  export ROS_SECURITY_LOG_FILE="${DIR}/log/ros-security.log"
```
Comment (jlaitine) [inline packaging/systemd/setup_fog.sh:17]: who writes the security log? if it is the ros nodes, they don't have rights to write into /enclave/ , as they are being executed as user "sad"

**Review by jlaitine — APPROVED** [review]

**Comment by quinz** [comment]
setup_fog.sh no longer exist. Will see how to include SROS setup in the new scripts once we have the tests and setup ready with latest fast DDS codes.

**Pull request closed without merging**